certified-connectors/PowerAppsCards/apiProperties.json

```diff
@@ -0,0 +1,50 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oauthSettings": {
+          "identityProvider": "aadcertificate",
+          "properties": {
+            "IsFirstParty": "True"
+          },
+          "scopes": [],
+          "clientId": "<<Enter client id>>",
```

Comment:
Where do we get the clientId?

Reply:
need to pass the value while registering connector